build.gradle

```diff
@@ -0,0 +1,129 @@
+apply plugin:"com.bertramlabs.asset-pipeline"
+apply plugin:"org.grails.grails-gsp"
+
+repositories {
+    mavenLocal()
+    maven { url "http://nexus.ala.org.au/content/groups/public/" }
+    maven { url "https://repo.grails.org/grails/core" }
+}
+
+configurations {
+    developmentOnly
+    runtimeClasspath {
+        extendsFrom developmentOnly
+    }
+}
+
+dependencies {
+    developmentOnly("org.springframework.boot:spring-boot-devtools")
+    compile "org.springframework.boot:spring-boot-starter-logging"
+    compile "org.springframework.boot:spring-boot-autoconfigure"
+    compile "org.grails:grails-core"
+    compile "org.springframework.boot:spring-boot-starter-actuator"
+    compile "org.springframework.boot:spring-boot-starter-tomcat"
+    compile "org.grails:grails-web-boot"
+    compile "org.grails:grails-logging"
+    compile "org.grails:grails-plugin-rest"
+    // compile "org.grails:grails-plugin-databinding"
+    compile "org.grails:grails-plugin-i18n"
+    compile "org.grails:grails-plugin-services"
+    compile "org.grails:grails-plugin-url-mappings"
+    compile "org.grails:grails-plugin-interceptors"
+    compile "org.grails.plugins:cache"
+    compile "org.grails.plugins:async"
+    //compile "org.grails.plugins:scaffolding"
+    compile "org.grails.plugins:events"
+    // compile "org.grails.plugins:hibernate5"
+    // compile "org.hibernate:hibernate-core:5.4.18.Final"
+    compile "org.grails.plugins:gsp"
+    compileOnly "io.micronaut:micronaut-inject-groovy"
+    console "org.grails:grails-console"
+    profile "org.grails.profiles:web"
+    runtime "org.glassfish.web:el-impl:2.1.2-b03"
+    //runtime "com.h2database:h2"
+    //runtime "org.apache.tomcat:tomcat-jdbc"
+    runtime "javax.xml.bind:jaxb-api:2.3.1"
+    runtime "xml-apis:xml-apis:1.4.01"
+    runtime "com.bertramlabs.plugins:asset-pipeline-grails:3.2.4"
+    testCompile "io.micronaut:micronaut-inject-groovy"
+    //testCompile "org.grails:grails-gorm-testing-support"
+    testCompile "org.mockito:mockito-core"
+    testCompile "org.grails:grails-web-testing-support"
+    testCompile "org.grails.plugins:geb"
+    testCompile "org.seleniumhq.selenium:selenium-remote-driver:3.14.0"
+    testCompile "org.seleniumhq.selenium:selenium-api:3.14.0"
+    testCompile "org.seleniumhq.selenium:selenium-support:3.14.0"
+    testRuntime "org.seleniumhq.selenium:selenium-chrome-driver:3.14.0"
+    testRuntime "org.seleniumhq.selenium:selenium-firefox-driver:3.14.0"
+
+    // External config
+    compile 'org.grails.plugins:external-config:2.0.0'
+
+    // Javascript libraries
+    // compile 'org.webjars:leaflet:1.6.0'
+    compile 'org.webjars:knockout:3.5.1'
+    // compile 'org.webjars:swagger-ui:3.18.2'
+
+    // ALA Plugins
+    compile group: 'org.grails.plugins', name: 'ala-auth', version: '3.2.2'
+    compile group: 'org.grails.plugins', name: 'ala-admin-plugin', version: '2.2'
+    compile group: 'org.grails.plugins', name: 'ala-bootstrap3', version: '3.2.2'
```

Comment:
@charvolant Could you bump this to 3.2.3 to pickup the newly released version that we are deploying for https://github.com/AtlasOfLivingAustralia/ala-infrastructure/issues/756 ?

Reply:
Upgraded